tsconfig.eslint.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    // ensure that nobody can accidentally use this config for a build
```

Comment:
i guess we can put comments into json without breaking it now? this template seems to do that a lot